about.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html> 
```

Comment:
`index.html` is the only page that needs to be moved out to the root for github pages deployment. The other pages could could remain in the pages folder. If we had more control over the github pages server, we might configure it to serve the `index.html` page even if it remained in the pages folder, though this could make building our link paths a little confusing. But since we don't have that control, we do have to conform to the expectations that github pages have.